### ISSUE TITLE
Create API to expose transcripts 

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH-MLA/AAPB2#1200]
**Done when** 
- [x] Provide an endpoint that exposes transcripts
- [x] Adhere to annotations that describe access levels for transcripts (which may differ from media)

**Comment by afred** [comment]
Questions:
  - [ ] Do we expect all transcripts to be _stored_ (on our end, in Solr) in a uniform serialization format? I.e. JSON, XML, SRT, other?
  - [ ] Do we expect all transcripts to be _delivered_ (via the API) in a uniform serialization format? I.e. JSON, XML, SRT, other?

**Comment by caseyedavis12** [comment]
@afred:

1a. Most transcripts will be in json (generated by Pop Up Archive) and stamped at the phrase level (every 5-7 seconds)
1b. Some materials will be .srt files created by the vendor as part of the digitization process and extraction of closed captioning. 
1c. Some transcripts will be sourced from Lexis Nexis for the NewsHour project, and I believe these are text files. 
1d. Finally, in the future, we will have transcripts created by kaldi. The output is word-stamped json.

2. That would be great if it's possible!

3. This is a good question. Let me confer with a computational humanist who just reached out to me to see what types of possible queries scholars might expect.

**Comment by caseyedavis12** [comment]
@caseyedavis12 and @sroosa to identify expected batches of transcripts that people will want.

**Comment by caseyedavis12** [comment]
Updates:

1. The API should expose only transcripts for assets in the Online Reading Room
2. We need to provide unique authentication credentials for approved API users (approval will be determined by the Project Manager after the user has agreed to the API Rules of Use shared with the user via email)

**Comment by jasoncorum** [comment]
@afred - For this update, "We need to provide unique authentication credentials for approved API users (approval will be determined by the Project Manager after the user has agreed to the API Rules of Use shared with the user via email)," it seems like we'd need to store keys and tokens in a database which also means we'd probably need to start keeping backups of the DB as well. Any additional thoughts?

**Comment by jasoncorum** [comment]
Dependent on PR for #1096.

**Comment by caseyedavis12** [comment]
@afred, @jasoncorum: I understand that previously we requested a transcript API that only exposes transcripts for assets currently in the Online Reading Room. Is it possible to also provide an API for all transcripts so that our stations can get copies of their transcripts? We still want the limited API access for only ORR transcripts, but we also need an endpoint for all transcripts.

**Comment by afred** [comment]
@caseyedavis12 this ticket as originally scoped is done. Rather than re-opening it for a new transcript API feature, can we create a new ticket, if the feature request is still relevant? Thanks!
